pkg/rpcclient/wsclient_test.go

```diff
@@ -296,10 +303,6 @@ func TestWSExecutionVMStateCheck(t *testing.T) {
 }
 
 func TestWSFilteredSubscriptions(t *testing.T) {
-	bCh := make(chan *block.Block)
-	txCh := make(chan *transaction.Transaction)
-	aerCh := make(chan *state.AppExecResult)
-	ntfCh := make(chan *state.ContainedNotificationEvent)
```

Comment:
Oh-oh, these channels are used by other tests in this suite. Thus, tests are failed to build.

Reply:
Fixed.